api/Makefile

```diff
@@ -81,6 +81,20 @@ version:
 	$(eval VERSION=$(if $(OVERWRITE_VERSION),$(OVERWRITE_VERSION),v$(shell ../scripts/vertagen/vertagen.sh)))
 	@echo "turing-api version:" $(VERSION)
 
+# ==================================
+# E2E tests
+# ==================================
+
+.PHONY: setup_e2e
+setup_e2e:
```

Comment:
Nit: Can the targets' names use hyphen as a separator as opposed to `_`, for consistency ?